mulighetsrommet-api/src/test/kotlin/no/nav/mulighetsrommet/api/services/TiltaksgjennomforingServiceTest.kt

```diff
@@ -0,0 +1,70 @@
+package no.nav.mulighetsrommet.api.services
+
+import io.kotest.core.spec.style.FunSpec
+import io.kotest.core.test.TestCaseOrder
+import no.nav.mulighetsrommet.api.repositories.AnsattTiltaksgjennomforingRepository
+import no.nav.mulighetsrommet.api.repositories.TiltaksgjennomforingRepository
+import no.nav.mulighetsrommet.api.repositories.TiltakstypeRepository
+import no.nav.mulighetsrommet.database.kotest.extensions.FlywayDatabaseTestListener
+import no.nav.mulighetsrommet.database.kotest.extensions.createApiDatabaseTestSchema
+import no.nav.mulighetsrommet.domain.dbo.TiltaksgjennomforingDbo
+import no.nav.mulighetsrommet.domain.dbo.TiltakstypeDbo
+import org.assertj.db.api.Assertions.assertThat
+import org.assertj.db.type.Table
+import org.junit.jupiter.api.Assertions.*
+import java.util.*
+
+class TiltaksgjennomforingServiceTest : FunSpec({
+    testOrder = TestCaseOrder.Sequential
+
+    val database = extension(FlywayDatabaseTestListener(createApiDatabaseTestSchema()))
+
+    beforeSpec {
+        val tiltaksgjennomforingRepository = TiltaksgjennomforingRepository(database.db)
+        val tiltakstypeReposity = TiltakstypeRepository(database.db)
+        val tiltakstypeId = UUID.fromString("0c565576-6a74-4bc2-ad5a-765580014ef9")
+
+        tiltakstypeReposity.upsert(
+            TiltakstypeDbo(
+                tiltakstypeId,
+                "",
+                ""
+            )
+        )
+
+        tiltaksgjennomforingRepository.upsert(
+            TiltaksgjennomforingDbo(
+                id = UUID.fromString("046b57eb-1c7e-4165-ac5d-39ad21ebf4ee"),
+                navn = null,
+                tiltakstypeId = tiltakstypeId,
+                tiltaksnummer = "",
+                virksomhetsnummer = null,
+                fraDato = null,
+                tilDato = null,
+                enhet = ""
+            )
+        )
+    }
+
+    context("TiltaksgjennomforingService") {
+        val tiltaksgjennomforingRepository = TiltaksgjennomforingRepository(database.db)
+        val ansattTiltaksgjennomforingRepository = AnsattTiltaksgjennomforingRepository(database.db)
+        val service = TiltaksgjennomforingService(tiltaksgjennomforingRepository, ansattTiltaksgjennomforingRepository)
+
+        test("Insert favoritt i liste") {
+            val table = Table(database.db.getDatasource(), "ansatt_tiltaksgjennomforing")
+
+            service.lagreGjennomforingTilMinliste("046b57eb-1c7e-4165-ac5d-39ad21ebf4ee", "1")
+            assertThat(table).row(0).column("navident").value().isEqualTo("1")
```

Comment:
Det finnes en utility på `FlywayDatabaseTestListener`-extensionen nå som gjør dette hakket enklere, slik at vi ikke eksplisitt trenger å opprette et AssertJ.Table: `database.assertThat("ansatt_tiltaksgjennomforing").row(0)...`

Her kan forøvrig også begge assertions forenkles til én assertion :smile: :
```
assertThat("ansatt_tiltaksgjennomforing")
    .row()
    .value("navident").isEqualTo("1")
    .value("tiltaksgjennomforing_id").isEqualTo("...")
```

Reply:
Ah, nice! Skal få oppdatert